### PR TITLE
Feature/startapp

### PR DESCRIPTION
**Pull request by suvda**

### CODE REVIEW

File: src/widgets/cards/cards.scss [hunk 1]
```diff
@@ -157,4 +158,24 @@ body {
...
+    &:after {
+      content: 'No results'
+    }
+  }
+
+  &.no-result #no-result-message {
+    display: block;
+  }
+
+  &.no-result.no-docs #no-result-message:after {
```
Comment (swarajban) [inline src/widgets/cards/cards.scss]: These can be nested

File: src/widgets/searchInput/searchInput.js [hunk 2]
```diff
@@ -42,6 +56,22 @@
       textElement.querySelector('span').focus();
     });
 
+    textElement.addEventListener('focusin', function (e) {
+      textElement.classList.add('editing');
```
Comment (swarajban) [inline src/widgets/searchInput/searchInput.js]: Why not use jQuery here? You're using it elsewhere

**Comment by swarajban** [comment]
made a few minor comments. other than that, looks good

File: src/widgets/searchInput/searchInput.js (hunk 2 above)
Comment (jagill) [inline src/widgets/searchInput/searchInput.js]: searchInput.js (and this SearchInput widget) doesn't use jQuery, has no jQuery dependency.

File: src/widgets/searchInput/searchInput.js [hunk 3]
```diff
@@ -53,6 +83,14 @@
       }
     );
 
+    var $warningMessage = $('<p>', {id: 'warning-message'});
```
Comment (swarajban) [inline src/widgets/searchInput/searchInput.js]: @jagill what about here?

File: src/widgets/searchInput/searchInput.scss [hunk 4]
```diff
@@ -102,14 +104,65 @@ $icon-size: 32px;
...
+      &.editing span {
+        background-color: #eee;
+      }
+    }
+
+    .mindmeld-message {
+      display: none;
+      position: relative;
+      top: $search-border-width;
+      $search-text-left: 30px;
```
Comment (jagill) [inline src/widgets/searchInput/searchInput.scss]: `$search-text-left` is already defined at line 82; you should use that definition, promoting it to a great scope if necessary.

File: src/widgets/searchInput/searchInput.js (hunk 3 above)
Comment (jagill) [inline src/widgets/searchInput/searchInput.js]: Good catch.  jshint is complaining about undefined "$".  @suvda lines 86-91 should all be done without jQuery.

File: src/widgets/searchInput/searchInput.scss (hunk 4 above)
Comment (suvda) [inline src/widgets/searchInput/searchInput.scss]: fixed

File: src/widgets/searchInput/searchInput.js (hunk 3 above)
Comment (suvda) [inline src/widgets/searchInput/searchInput.js]: fixed

File: src/widgets/cards/cards.scss (hunk 1 above)
Comment (suvda) [inline src/widgets/cards/cards.scss]: fixed

File: src/widgets/searchInput/searchInput.js (hunk 2 above)
Comment (suvda) [inline src/widgets/searchInput/searchInput.js]: leave as is

**Pull request merged**